.github/workflows/functional-tests.yml

```
@@ -0,0 +1,32 @@
+name: Functional Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
```

How about we run these on merge too, to match the other workflows?

```suggestion
  pull_request:
  push:
    branches:
      - dev
```